Core/AppRuntime/Source/AppRuntimeJavaScriptCore.cpp

```diff
@@ -1,3 +1,4 @@
+#include "WorkQueue.h"
```

Comment:
Why do you need to include WorkQueue? Also, the first include should be the AppRuntime.